charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml

```diff
@@ -7,8 +7,28 @@ metadata:
     {{- include "newrelic.common.labels" . | nindent 4 }}
 data:
   deployment-config.yaml: |
-    {{- with include "nrKubernetesOtel.deployment.configMap.config" . }}
-    {{- . | nindent 4 }}
+    {{- if .Values.otel }}
+    receivers:
+      {{- include "deployment-receivers" . | nindent 6 }}
```

Comment:
If we're sticking with the deployment, daemonset, shared paradigm, all sections should include a way to include shared components.